lib/services/android-debug-service.ts

```diff
@@ -177,6 +178,12 @@ class AndroidDebugService implements IDebugService {
 			this.$devicesService.execute(action).wait();
 		}).future<void>()();
 	}
+	
+	private sleep(ms: number): void {
+		let fiber = Fiber.current;
```

Comment:
hm, is there a way to get here without fiber? In this case Fiber.current will be undefined